artemis-hawtio/activemq-branding/src/main/webapp/plugin/js/plugin.js

```diff
@@ -59,10 +59,14 @@ var activemqBranding = (function (self) {
 
     self.module = angular.module(self.pluginName, ['hawtioCore']);
     self.module.run(function (branding) {
-        self.log.debug("ActivMQ theme loaded");
+        self.log.info("ActiveMQ theme loaded");
+        if (localStorage['theme'] != 'activemq' || localStorage['branding'] != 'activemq') {
+            localStorage['theme'] = 'activemq';
+            localStorage['branding'] = 'activemq';
+            location.reload();
```

Comment:
There are checks in this file to set the branding/theme if it's not already set.  Any reason why you didn't just remove the if statements?

```js
    if (!localStorage['theme'])         if (!('theme' in localStorage)) {
            localStorage['theme'] = 'activemq';
    }
```

Reply:
If anything those existing if and set can be removed entirely. The set should be within the run from what I understood debugging. Because I couldn't recreate though I didn't want to remove in case of some other side effect.

Reply:
@mtaylor if you have reproducer we can remove and you can retest to check?

Reply:
why you need ifs at all? why not always do

localStorage['theme'] = 'activemq';
location.reload();

Reply:
to avoid reload causing it to end up in a reload loop

Reply:
got it